tests/unit/error/merr_test.c

```diff
@@ -53,8 +53,12 @@ MTF_DEFINE_UTEST(hse_err_test, merr_test_1)
     ASSERT_EQ(0, strcmp(errinfo, "success"));
 
     rval = EINVAL;
+    // clang-format off
+    // - cannot let clang-format add or remove newlines
```

Comment:
Would this actually happen given our new configuration?

Reply:
Builds fail if these files are out of order.

Reply:
Sure, but I am asking if the format file would actually change this.

Reply:
Best not to leave it to chance, cuz who knows what the next version of clang-format will do?

Reply:
It did in one iteration.  Not sure now, but I'm with @beckerg.  I spent a few minutes trying to make it work in either order, but I gave up in frustration.

Reply:
We know the next version of clang format will fail a PR if it Fs with this.

Reply:
And it'll be a grand waste of time looking into it...